packages/flutter/test/material/card_test.dart

```diff
@@ -11,6 +11,21 @@ import 'package:flutter_test/flutter_test.dart';
 import '../widgets/semantics_tester.dart';
 
 void main() {
+  Material _getCardMaterial(WidgetTester tester) {
```

Comment:
Utility functions that are just used by one test can just be defined as local functions with the test.